examples/tls/BasicTLSClient.cpp

```diff
@@ -23,9 +23,12 @@ int main() {
     hazelcast::client::address serverAddress("127.0.0.1", 5701);
     config.get_network_config().add_address(serverAddress);
 
+    boost::asio::ssl::context ctx(boost::asio::ssl::context::method::tlsv12_client);
+    ctx.set_default_verify_paths();
+    ctx.load_verify_file("/path/to/my/server/public/certificate");
```

Comment:
Should we also do `ctx.set_verify_mode(ssl::verify_peer)` to make sure the certificate is actually verified? What is the default behavior if you don't set the verify mode?

Reply:
Added.